apps/hestia/src/components/cexOnRamp/template.tsx

```diff
@@ -0,0 +1,122 @@
+"use client";
+
+import { renderSendWidget } from "@cedelabs/widgets-universal";
+import { useConnectWalletProvider } from "@orderbook/core/providers/user/connectWalletProvider";
+import { Typography } from "@polkadex/ux";
+import { RiInformation2Line } from "@remixicon/react";
+import { Fragment, useEffect, useMemo, useRef } from "react";
+import { useResizeObserver, useWindowSize } from "usehooks-ts";
+
+import { Help } from "../transfer/Help";
```

Comment:
We are using component based project structure so all the components required for a page, should be in current folder only. 

For e.g. You can't import `Help` component from transfer page. I would suggest you to create a Help component in current directory.